static/js/publisher/tour/tourOverlay.js

```diff
@@ -48,18 +48,41 @@ const getMaskFromRect = rect => {
   };
 };
 
-// find DOM element for each step, ignore steps with no element
+// get mask that is an union of all elements' masks
+// calculates the rectangle that contains each individual element rectangles
+const getMaskFromElements = elements => {
+  return elements.map(el => getMaskFromRect(getRectFromEl(el))).reduce(
```

Comment:
This is a little bit hard to parse - map into two nested functions into reduce. I guess it's not so hard to understand what goes into this function and what comes out, but the internals are a bit hazy for me.

Reply:
Wanted to format it nicer for better readability, but prettier wants it all in one line ;)